src/lib/uploader.ts

```diff
@@ -135,6 +147,18 @@ export default class MinidumpUploader {
     });
   }
 
+  /**
+   * Flushes locally cached minidumps from the queue.
+   */
+  public async flushQueuedMinidumps(): Promise<void> {
+    const storedRequests = this.queue.get();
+    if (storedRequests.length) {
```

Comment:
Queue will always be an array, so there's no need to check if there's anything. It'll noop for empty list.